docs/guide/next-steps.md

```diff
@@ -0,0 +1,11 @@
+# Next Steps
+
+Now that you have a basic knowledge howto work with webcomponents there are some more in depts informations.
```

Comment:
"Now that you have a basic understanding of web components, you can find some more in depth information in these resources."